### PR TITLE
[improve][doc] SEO for Tiered Storage

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
This PR do SEO for Tiered Storage

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

image
image
image
image
image
image
image

### CODE REVIEW

File: docs/tiered-storage-overview.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 id: tiered-storage-overview
 title: Overview of tiered storage
 sidebar_label: "Overview"
+description: Get a comprehensive understanding of concepts and install methods of Pulsar tiered storage.
```
Comment (Anonymitaet) [inline docs/tiered-storage-overview.md:5]:
```suggestion
description: Get a comprehensive understanding of use cases, installation methods, and working principles of Pulsar tiered storage.
```

File: docs/tiered-storage-overview.md [hunk 2]
```diff
@@ -30,13 +31,15 @@ For example, if you have a topic containing user actions that you use to train y
 
 ## How to install tiered storage offloaders?
 
-Pulsar releases a separate binary distribution, containing the tiered storage offloaders. To enable those offloaders, you need to download the offloaders tarball release:
+Pulsar releases a separate binary distribution, containing the tiered storage offloaders. To enable those offloaders, you need to complete the following steps.
+
+1. download the offloaders tarball release
```
Comment (Anonymitaet) [inline docs/tiered-storage-overview.md:36]:
```suggestion
1. Download the offloaders tarball release.
```

File: docs/tiered-storage-overview.md [hunk 3]
```diff
@@ -30,13 +31,15 @@ For example, if you have a topic containing user actions that you use to train y
...
 
 ```bash
 wget pulsar:offloader_release_url
 ```
 
-After you download the tarball, untar the offloaders package and copy the offloaders as `offloaders` in the pulsar directory:
+2. Untar the offloaders package and copy the offloaders as `offloaders` in the pulsar directory
```
Comment (Anonymitaet) [inline docs/tiered-storage-overview.md:42]:
```suggestion
2. Untar the offloaders package and copy the offloaders as `offloaders` in the pulsar directory.
```

**Comment by zuobiao-zhou** [comment]
Updated, PTAL @Anonymitaet

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**